### PR TITLE
Support firefox

### PR DESCRIPTION
**Pull request by banocean**

### CODE REVIEW

File: manifest.json [hunk 1]
```diff
@@ -13,7 +13,19 @@
...
+      "strict_min_version": "117.0"
+    },
+    "gecko_android": {
+      "id": "j.skup.test@gmail.com",
+      "strict_min_version": "117.0"
+    }
+  },
+
   "background": {
+    "scripts": ["apply.js"],
```
Comment (PanLiszka) [inline manifest.json:28]: all patches work :)

**Review by PanLiszka — APPROVED** [review]
LGTM! 😸

**Review by Marioneq4958 — COMMENTED** [review]
It works, but
obraz

**Comment by banocean** [comment]
> It works, but obraz

If we want to use a single manifest file (+ don't have a build step) for all browsers, then I can't do anything about it

**Pull request merged**